pkg/porter/explain.go

```diff
@@ -494,10 +501,18 @@ func (p *Porter) printInstallationInstructionBlock(bun *PrintableBundle, bundleR
 
 	var credentialFlags string
 	if len(bun.Credentials) > 0 {
-		credentialFlags += "--cred mycreds"
+		credentialFlags += " --cred mycreds"
 	}
 
-	fmt.Fprintf(p.Out, "porter install%s%s%s\n", bundleReferenceFlag, requiredParameterFlags, credentialFlags)
+	porterInstallCommand := fmt.Sprintf("porter install%s%s%s", bundleReferenceFlag, requiredParameterFlags, credentialFlags)
+
+	// Check whether the bundle requires docker mixin and add flag for host access for install command.
```

Comment:
This comment is a bit misleading since we aren't checking for the docker mixin, but if the bundle requires the docker socket to be mounted into the bundle.